server/controllers/gift.controller.js

```diff
@@ -30,7 +29,7 @@ module.exports = {
     let giftParams = Object.assign({}, req.body, {event_id: req.params.id});
```

Comment:
it is just a temporary value, changing with every request, so there is no need for it to be declared with const